app/api/resources/user.py

```diff
@@ -276,11 +279,22 @@ def post(cls):
         if is_valid != {}:
             return is_valid, 400
 
-        result = DAO.create_user(data)
-
-        if result[1] is 200:
+        failed_send = False
+        try:
             send_email_verification_message(data["name"], data["email"])
 
+        except SMTPException as se:
+            failed_send = True
+            print("SMTP error: {0}".format(se))
+            return messages.SMTP_ERROR, 400
```

Comment:
Can you please use a HTTPStatus code, like it is in the HTTPStatus module (mentioned [Coding Standards](https://github.com/anitab-org/mentorship-backend/wiki/Coding-Standards) wiki page)

Reply:
I've changed the UserRegister response code to HTTPStatus code and import the library on the top of the file. I didn't do this in my previous commit as it was done before issue #519 and it was consistent to how we did it before the requested issue. I changed only on the UserRegister class not the rest of the file so Nandini will have less merge conflict when PR #532 is being merged. 😉